couchpotato/core/notifications/emby.py

```diff
@@ -0,0 +1,89 @@
+import json
+import urllib, urllib2
+
+from couchpotato.core.helpers.variable import cleanHost
+from couchpotato.core.logger import CPLog
+from couchpotato.core.notifications.base import Notification
+
+
+log = CPLog(__name__)
+
+autoload = 'Emby'
+
+
+class Emby(Notification):
+
+    def notify(self, message = '', data = None, listener = None):
+        host = self.conf('host')
+        apikey = self.conf('apikey')
+
+        host = cleanHost(host)
+        url = '%semby/Library/Series/Updated' % (host)
```

Comment:
From checking the API code there currently isn't, but I'm sure @LukePulverenti would be willing to make changes/add to the API.
https://github.com/MediaBrowser/Emby/blob/master/MediaBrowser.Api/Library/LibraryService.cs

In my testing the API always triggers a full library scan, even for tvshows when a "tvdbid" is passed but hopefully this can be improved in the future on the Emby side.